config/config_test.go

```diff
@@ -10,10 +11,11 @@ import (
 
 func TestSpec(t *testing.T) {
 
-	cfg, err := config.Get()
-
 	Convey("Given an environment with no environment variables set", t, func() {
 
+		os.Clearenv()
```

Comment:
You need to be mindful when doing this in tests. If the tests are running in parallel this can create a race condition between different test cases accessing env vars. The result is inconsistent tests results and random failures.

I don't know if this is an issue here but it's something to bear in mind.